annotations.md

```diff
@@ -26,4 +26,6 @@ This specification defines the following annotation keys, intended for but not l
 * **org.opencontainers.image.revision** Source control revision identifier for packaged software.
 * **org.opencontainers.image.vendor** Name of the distributing entity, organization or individual.
 * **org.opencontainers.image.licenses** Comma-separated list of licenses under which contained software is distributed, in [SPDX Short identifier]https://spdx.org/licenses/] form.
-* **org.opencontainers.image.ref.name** Name of the reference for a target (string). SHOULD only be considered valid when on descriptors on `index.json` within [image layout](image-layout.md).
+* **org.opencontainers.image.ref.name** Name of the reference for a target (string).
+  * SHOULD only be considered valid when on descriptors on `index.json` within [image layout](image-layout.md).
+  * Character set of the value SHOULD conform to the [Uniform Resource Identifier (URI) RFC3986](https://tools.ietf.org/html/rfc3986) [`segment-nz` (from appendix A)](https://tools.ietf.org/html/rfc3986#appendix-A).
```

Comment:
The canonical link for `segment-nz` is [section 3.3][1], appendix A just collects the ABNF together for ease-of-use.  If you want to keep the appendix reference, maybe “from appendix A” → “see appendix A”?

Also, `segment-nz` is a bigger charset than [Docker's tags][1].

[1]: https://github.com/opencontainers/image-spec/pull/671#issuecomment-301079786

[1]: https://tools.ietf.org/html/rfc3986#section-3.3